ui/tract-renderer/src/main/java/org/cru/godtools/tract/ui/controller/ParentController.kt

```diff
@@ -37,17 +37,19 @@ abstract class ParentController<T> protected constructor(
     // endregion Lifecycle
 
     // region Child Content
-    protected abstract val contentContainer: LinearLayout
+    protected abstract val contentContainer: ViewGroup
     private val childCache by lazy { UiControllerCache(contentContainer, this) }
     private var children: List<BaseController<Content>>? = null
 
+    protected open val contentToRender get() = model?.content
```

Comment:
Is there value in extracting this to a variable?

Reply:
I override it in `FallbackController`, this allows rendering of fallbacks to only render the first element instead of rendering every element it contains.